plugins/modules/docker_compose_v2.py

```diff
@@ -0,0 +1,371 @@
+#!/usr/bin/python
+
+from __future__ import (absolute_import, division, print_function)
+from ansible.module_utils.basic import AnsibleModule
+import os
+from python_on_whales.docker_client import DockerClient
+from python_on_whales.exceptions import (
+    DockerException,
+)
+__metaclass__ = type
```

Comment:
Please move everything that's not `metaclass` and `__future__` below the documentation (as in all other modules).